### PR TITLE
OSOE-84: Improve documentation, fix broken link

### PR DESCRIPTION
**Pull request by 0liver**
[OSOE-84](https://lombiq.atlassian.net/browse/OSOE-84)

[OSOE-84]: https://lombiq.atlassian.net/browse/OSOE-84?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -104,11 +104,11 @@ export default {
 </script>
 ```
 
-You can include the `<vue-component area="My.Module name="my-article">` tag helper in your code. This will add Vue and _My.Module/wwwroot/vue/my-article.js_ to the resource manager (using the `vue-component-{name}` resource) as well as the `VueComponent-MyArticle` shape (the SFC's kebab-case name is converted into PascalCase). Include `depends-on="vue-component-my-article"` in your app's `<script>` element to ensure correct load order.
+You can include the `<vue-component area="My.Module" name="my-article">` tag helper in your code. This will add Vue and _My.Module/wwwroot/vue/my-article.js_ to the resource manager (using the `vue-component-{name}` resource) as well as the `VueComponent-MyArticle` shape (the SFC's kebab-case name is converted into PascalCase). Include `depends-on="vue-component-my-article"` in your app's `<script>` element to ensure correct load order.
```
Comment (0liver) [inline Readme.md:107]: @DAud-IcI Added a `"` after `area="My.Module`, same below.

File: Readme.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Do you want to quickly try out this project and see it in action? Check it out i
 
 ## Prerequisites
 
-1. This project relies on [Lombiq Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions/), please see its pre-requisites [here](https://github.com/Lombiq/NodeJs-Extensions/#pre-requisites).
+1. This project relies on [Lombiq Node.js Extensions](https://github.com/Lombiq/NodeJs-Extensions/), please see its prerequisites [here](https://github.com/Lombiq/NodeJs-Extensions/#prerequisites).
```
Comment (adityaraute) [inline Readme.md:13]: Why is this change necessary? Can be avoided

File: Readme.md [hunk 3]
```diff
@@ -46,21 +46,21 @@ If you just want to see the whole thing in action, check out the [Samples projec
 
 ## Using Vue.js node packages
 
-The packages will be automatically installed on build (i.e. `dotnet build`) or you can trigger it using the `npm install` command.
+The packages will be automatically installed on build (i.e. `dotnet build`). You can also trigger the installation manually using the `pnpm install` command.
```
Comment (adityaraute) [inline Readme.md:49]:
```suggestion
The packages will be automatically installed on build (i.e. `dotnet build`). You can also trigger the installation manually using the `npm install` command.
```

File: Readme.md (hunk 2 above)
Comment (sarahelsaig) [reply Readme.md:13]: What do you mean? Spelling should be  consistent across the document (and across our repos) so this is a good change.

File: Readme.md (hunk 3 above)
Comment (sarahelsaig) [reply Readme.md:49]: No, `pnpm` is correct. The Node.js Extensions parent library requires [PNPM](https://pnpm.io/), see:
https://github.com/Lombiq/NodeJs-Extensions/#prerequisites

**Review by sarahelsaig — APPROVED** [review]
